src/main/java/com/example/gradecalculator/controller/UserController.java

```diff
@@ -127,5 +125,20 @@ public String performLogout(Authentication authentication, HttpServletRequest re
         this.logoutHandler.logout(request, response, authentication);
         return "index";
     }
+
+    @DeleteMapping("/myProfile")
```

Comment:
route for editing is 'editProfile' so to be consistent wouldn't 'deleteProfile' be better?